### DISCUSSION TITLE
DataGridTextColumn bind to function

### DISCUSSION THREAD

**Discussion by Aytharn** [fsprojects/Avalonia.FuncUI#405] [General]
I am looking at https://github.com/fsprojects/Avalonia.FuncUI/blob/master/src/Examples/Examples.DataGridPlayground/Program.fs to see how to bind data to a grid, but I need to call functions here, since I have a functional i18n implemented, for instance I need to call Name(Lang.French) or Name(Settings.Lang) to get the proper version of a content to display: how can I do that in FuncUI ?

**Comment by JaggerJo** [comment]
You can use DataTemplates for the cells and call your i18n functions there.

**Comment by Aytharn** [comment]
DataGridTextColumn doesn't appear to have a template property... DataGrid has but it doesn't do anything: can you provide an example? Thanks!

**Reply by Numpsy** [reply]
Does it need to use ```DataGridTemplateColumn``` instead of ```DataGridTextColumn``` ?

**Reply by Aytharn** [reply]
I didn't know it existed...

**Reply by Numpsy** [reply]
I haven't used it myself, just saw the example at https://github.com/fsprojects/Avalonia.FuncUI/blob/8306b0169fdb98ac3bc98a76498afbbff420a0fd/src/Examples/Examples.DataGridPlayground/Program.fs#L61
